project-maintainers.csv

```diff
@@ -22,33 +22,56 @@ Graduated,Kubernetes,Antonio Ojea,Google,aojea,https://git.k8s.io/steering#membe
 ,,Adolfo García Veytia,Chainguard,puerco,
 ,,Carlos Tadeu Panato Jr.,Chainguard,cpanato,
 ,,Verónica López,PlanetScale,Verolop,
-Graduated,Prometheus,Arthur Sens Silva,Grafana Labs,ArthurSens,https://prometheus.io/governance/#team-members
+Graduated,Prometheus,Alex Greenbank,Grafana Labs,alexgreenbank,https://prometheus.io/governance/#team-members
+,,Arianna Vespri,,vesari,
+,,Arthur Sens Silva,Grafana Labs,ArthurSens,
+,,Arve Knudsen,Grafana Labs,aknuds1,
 ,,Augustin Husson,Amadeus,Nexucis,
+,,Ayoub Mrini,Red Hat,machine424,
 ,,Bartłomiej Płotka,Google,bwplotka,
 ,,Ben Kochie,Reddit,superq,
 ,,Ben Reedy,Indue Ltd,breed808,
 ,,Björn Rabenstein,Grafana Labs,beorn7,
 ,,Bryan Boreham,Grafana Labs,bboreham,
 ,,Calle Pettersson,Instabee Group,carlpett,
 ,,Callum Styan,Grafana Labs,cstyan,
-,,Sanket Kalekar,Freelancer,ooye-sanket,
+,,Carrie Edwards,Grafana Labs,carrieedwards,
 ,,Chris Marchbanks,Grafana Labs,csmarchbanks,
+,,Chris Sinjakli,PlanetScale,Sinjo,
+,,Conrad Hoffmann,,bitfehler,
+,,Cristian Greco,Grafana Labs,cristiangreco,
+,,Daniel Magliola,IndeedFlex,dmagliola,
+,,Daniel Swarbrick,,dswarbrick,
 ,,David Ashpole,Google,dashpole,
-,,Frederic Branczyk,Polar Signals,brancz,
+,,David Leadbeater,G-Research,dgl,
+,,Doug Hoard,Confluent,dhoard,
 ,,Fabian Stäber,Grafana Labs,fstab,
+,,Fiona Liao,Grafana Labs,fionaliao,
+,,Frederic Branczyk,Polar Signals,brancz,
 ,,Ganesh Vernekar,Grafana Labs,codesome,
+,,George Krajcsovits,Grafana Labs,krajorama,
 ,,Goutham Veeramachaneni,Grafana Labs,gouthamve,
+,,Gregor Zeitlinger,Grafana Labs,zeitlinger,
+,,Jan Fajerski,Red Hat,jan--f,
+,,Jan-Otto Kröpke,,jkroepke,

```

Comment:
```suggestion
,,Jan-Otto Kröpke, Independent,jkroepke,
```

Some entries does have Independent, others not.

Reply:
Yea, I make it consistently empty, but merging the entry you want, thanks!

Reply:
Please merge it. From my point of view, making it explicit clarifies the affiliate compared to leaving it empty.